examples/hal/quad/main.rs

```diff
@@ -449,15 +438,7 @@ fn main() {
             binding: 0,
             array_offset: 0,
             descriptors: Some(
-                pso::Descriptor::Image(&image_srv, i::Layout::Undefined)
-            ),
-        },
-        pso::DescriptorSetWrite {
-            set: &desc_set,
-            binding: 1,
-            array_offset: 0,
-            descriptors: Some(
-                pso::Descriptor::Sampler(&sampler)
+                pso::Descriptor::CombinedImageSampler(&image_srv, i::Layout::ShaderReadOnlyOptimal, &sampler)
```

Comment:
looks like we had a bug in here by specifying `Undefined`?

Reply:
I only changed it because `ShaderReadOnlyOptimal` is more optimal. No clue if it's a bug.